src/content_sao.cpp

```diff
@@ -1193,6 +1193,10 @@ void PlayerSAO::setPos(const v3f &pos)
 	if(isAttached())
 		return;
 
+	// Send mapblock of target location
+	v3s16 blockpos = v3s16(pos.X / MAP_BLOCKSIZE, pos.Y / MAP_BLOCKSIZE, pos.Z / MAP_BLOCKSIZE);
```

Comment:
`v3s16 blockpos =  floatToInt(pos, BS);`

Reply:
No! `BS` == 10. `MAP_BLOCKSIZE` == 16

Reply:
I don't think `floatToInt` is the correct conversion either here.

Reply:
Oops, you're totally right.